ct-app/economic_handler/economic_handler.py

```diff
@@ -219,7 +219,7 @@ async def get_unique_safe_peerId_links(self):
         Returns a dictionary containing all unique
         source_peerId-source_address links
         """
-        response = await self.api.get_unique_safe_peerId_links()
+        response = await self.api.unique_safe_peerId_links()
```

Comment:
Note that the function name changed to get_unique_nodeAddress_peerId_aggbalance_link() in one of my recent PRs already merged to development. Consider this when resolving the merge conflicts :)

Reply:
resolved!